README.md

```diff
@@ -25,6 +26,15 @@ To install Bun, run the following command in your terminal:
 curl -fsSL https://bun.sh/install | bash
 ```
 
+On Windows:
```

Comment:
boi, please move this under the #installation section, before or after the MacOS instructions